pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -344,6 +345,40 @@ def load_triggers(input_file, vetoes):
     return trigs
 
 
+def load_trig_data(input_file, ifos, vetoes):
+    """Loads triggers data and ifo triggers data into a dictionary from
+    a PyGRB bank file"""
+    trigs = load_triggers(input_file, vetoes)
+    # Load network data into dictionaty
+    data = {
+        "end_time": trigs["/network/end_time_gc"][...],
+        "ra": trigs["/network/ra"][...],
+        "dec": trigs["/network/dec"][...],
+        "coherent_snr": trigs["/network/coherent_snr"][...],
+        "null_snr": trigs["/network/null_snr"][...],
+        "reweighted_snr": trigs["/network/reweighted_snr"][...],
+        "chisq": trigs["/network/my_network_chisq"][...],
+        "bank_chisq": trigs["/network/my_network_bank_chisq"][...],
+        "cont_chisq": trigs["/network/my_network_cont_chisq"][...],
+        "time_slide_id": trigs["/network/slide_id"][...],
+        "mass1": trigs["/network/mass1"][...],
+        "mass2": trigs["/network/mass2"][...],
+        "mchirp": trigs["/network/mchirp"][...],
```

Comment:
I'm not sure we want to load everything into a dictionary. One of the nice things about HDF5 files is we can access their contents easily without having to load everything at once.

Reply:
I agree, but I think that having a function in a single place is much easier to maintain for future code upgrades, and the loss of performance is not that huge.